src/shared/components/oncoprint/controls/CustomDropdown.tsx

```diff
@@ -6,7 +6,7 @@ import {observable} from "mobx";
 
 export interface ICustomDropdownProps extends ButtonProps {
     title: string;
-    buttonProps?: ButtonProps;
+    classType?: string;
```

Comment:
I would keep things consistent/generic and make this "className".  You can then use the classNames library to apply both dropdown-menu and anything passed in through this prop.   Search for "classNames" and you'll find it used a lot in our project

Reply:
I have rename this to `classsName`. I have used the _classNames_ lib as per your suggestion. I am unfamiliar with React and your suggestion:
>You can then use the classNames library to apply both dropdown-menu and anything passed in through this prop.

may mean there are more clever ways of class propagation to children with _classNames_. If so, I am more than interested to be pointed in the right direction.

Reply:
no, you have it right now.